### DISCUSSION TITLE
Mouse pad

### DISCUSSION THREAD

**Discussion by volfpeter** [elementary/store#116] [Product Ideas]
Probably any mouse pad would do.
Personally, I've been on the lookout for a decent/clean looking ultra-thin one (0.4-0.6mm) that would fit between the keyboard and the screen of my notebook, so I could easily carry it around while also protecting the screen.
On a similar note, a microfiber cloth that I could use to clean my notebook could also be useful.

**Comment by cassidyjames** [comment]
Thanks for the suggestion! We partner with Printful for our products since they can produce them on demand with no waste and no order minimums. However, it means we're also limited by their [catalog of products](https://www.printful.com/custom-products). I don't see mousepads or microfiber cloths at this time.

**Reply by volfpeter** [reply]
Ah, I see, I didn't know how it works. Thanks!
